docs/sandbox/pairing-actions.md

```diff
@@ -0,0 +1,95 @@
+# _Sharly Chess_ - Pairing actions
+
+This page provides an overview of the actions in the tournaments, players, and pairings tabs.
+
+## Round status definition
+
+| Round                | Definition                                                                                | FR                                 |
+|----------------------|-------------------------------------------------------------------------------------------|------------------------------------|
+| Current round        | The last round with pairings                                                              | Ronde courante (ou ronde en cours) |
+| Previous round       | The round immediately preceding the current round (N-1)                                   | Ronde précédente                   |
+| Past rounds          | The rounds preceding the previous round (N-2, N-3, ...)                                   | Rondes passées                     |
+| Next round           | The round immediately following the current round (N+1)                                   | Ronde suivante                     |
+| Future rounds        | The rounds following the next round (N+2, N+3, ...)                                       | Rondes futures                     |
+| Last published round | The last round for which the chief arbiter changed the status from "draft" to "published" | Dernière ronde publiée             |
+
+> [!NOTE]
+> :information_source:
+> - The concept of **last published round** does not yet exist and will be implemented when Access storage is discontinued.
+> In the current version, all rounds with pairings are considered published (draft status does not exist).
+> - The concepts in the table below are tournament-specific. The concept of a displayed or selected round is relative to the web interface; it is the active round in the Pairings tab.
+
+## Tournament modifications
+
+To be completed (some tournament modifications should not be allowed after a tournament has started).
+
+## Player Modifications (Elo, FIDE title, or name)
+
+> [!TIP]
+> :point_right: Changes to player information other than Elo, FIDE title, or name have no impact on pairings and ratings.
+
+> [!NOTE]
+> :information_source:
+> - _Papi_ recalculates pairing numbers every round (contrary to FIDE regulations).
+> - Within a certain period of time after the publication of the results of round N (or before publication of the results), the change takes effect for the ratings of round N and the pairings of round N+1.
+> After the publication of round N+1, the ratings are changed starting with round N+2.
+
+| Time                                                           |  FIDE Authorized   |  Registration  | Recalculation<br/>numbers<br/>pairings |      Modification<br/>ratings       |
```

Comment:
"Recalculation numbers pairings" sounds weird.
I'd like @timothyarmes's opinion on that

Reply:
Yes, that's wrong. I suggest "Round to recalculate pairings"